models/user.go

```diff
@@ -15,10 +15,15 @@ func init() {
 	orm.RegisterModel(new(User))
 }
 
+// TableName set the custom table name to plural because the default table name is singular
+func (u *User) TableName() string {
+	return "users"
+}
```

Comment:
Added this to [set custom table name for the model](https://beego.me/docs/mvc/model/models.md#custom-table-name)